### PR TITLE
Extend bounds system to finite volume

### PR DESCRIPTION
**Pull request by GiudGiud**
Kind of prospective but also fixes a potential seg fault

### CODE REVIEW

File: test/tests/auxkernels/bounds/constant_bounds_fv.i [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  xmin = 0
+  xmax = 1
+  nx = 10
+[]
+
+[Variables]
+  [u]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
```
Comment (lindsayad) [inline test/tests/auxkernels/bounds/constant_bounds_fv.i:13]:
```suggestion
  type = MooseVariableFVReal
```

File: test/tests/auxkernels/bounds/constant_bounds_fv.i [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  []
+  [v]
+    order = CONSTANT
+    family = MONOMIAL
```
Comment (lindsayad) [inline test/tests/auxkernels/bounds/constant_bounds_fv.i:17]:
```suggestion
```

File: test/tests/auxkernels/bounds/constant_bounds_fv.i [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    type = MooseVariableFVReal
+  []
+[]
+
+[AuxVariables]
+  [bounds_dummy]
+    order = CONSTANT
+    family = MONOMIAL
+    # fv = true
```
Comment (lindsayad) [inline test/tests/auxkernels/bounds/constant_bounds_fv.i:26]:
```suggestion
```

File: test/tests/auxkernels/bounds/constant_bounds_fv.i [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    variable = bounds_dummy
+    bounded_variable = v
+    bound_type = lower
+    bound_value = -1
+  []
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = 'PJFNK'
```
Comment (lindsayad) [inline test/tests/auxkernels/bounds/constant_bounds_fv.i:111]:
```suggestion
  solve_type = 'NEWTON'
```

File: test/tests/auxkernels/bounds/tests [hunk 5]
```diff
@@ -26,6 +26,15 @@
...
     requirement = "The system shall include the ability to compute a constant monomial field variable with constant upper and lower bounds."
   []
+  [constant_bounds_fv]
+    type = 'Exodiff'
+    input = 'constant_bounds_fv.i'
+    exodiff = 'constant_bounds_fv_out.e'
+    scale_refine = 3
+    design = 'ConstantBoundsAux.md'
+    issues = '#951 #14946'
+    requirement = "The system shall include the ability to compute a finite volume variable with constant upper and lower bounds."
```
Comment (lindsayad) [inline test/tests/auxkernels/bounds/tests:36]:
```suggestion
    requirement = "The system shall be able to impose constant upper and lower bounds on a finite volume variable."
```

**Review by lindsayad — CHANGES_REQUESTED** [review]

File: test/tests/auxkernels/bounds/constant_bounds_fv.i (hunk 1 above)
Comment (GiudGiud) [reply test/tests/auxkernels/bounds/constant_bounds_fv.i:13]: was kind of doing this on purpose to make sure we re resilient to other syntax

Comment (lindsayad) [reply test/tests/auxkernels/bounds/constant_bounds_fv.i:13]: ok that's fine

**Review by lindsayad — DISMISSED** [review]

**Comment by GiudGiud** [comment]
@lindsayad 

I had forgotten the ad_indexing_type tests clause

**Review by lindsayad — APPROVED** [review]

**Pull request merged**